Resources/Prototypes/_White/Catalog/uplink_catalog.yml

```
@@ -50,4 +50,14 @@
\ No newline at end of file
+  saleLimit: 1
+
+- type: listing
+  id: UplinkImplanterSyndi
+  name: uplink-implanter-name
+  description: uplink-implanter-desc
+  productEntity: ImplanterSyndi
+  cost:
+    Telecrystal: 1
+  categories:
+  - UplinkImplants
```

yamllint

[warning] wrong indentation: expected 4 but found 2

(indentation)